test/e2e/network_test.go

```diff
@@ -70,6 +70,17 @@ func testEastWestNetworking(t *testing.T) {
 		err = testCtx.waitUntilJobSucceeds(linuxCurlerJob.Name)
 		assert.NoError(t, err, "could not curl the Windows server from a linux container")
 
+		//Create a ClusterIP service
+		clusterIPSVC, err := testCtx.createService(winServerDeployment.Name, v1.ServiceTypeLoadBalancer, *winServerDeployment.Spec.Selector)
```

Comment:
shouldn't this be `v1.ServiceTypeClusterIP`?

Reply:
Good catch, fixed